draft-ietf-jsonpath-base.md

```diff
@@ -180,15 +180,15 @@ Additional terms used in this specification are defined below.
 
 Value:
 : As per {{-json}}, a structure conforming to the generic data model of JSON, i.e.,
-  composed of components such as structured values, namely JSON objects and arrays, and
-  primitive data, namely numbers and text strings as well as the special
+  composed of structured values, namely JSON objects and arrays, and
+  primitive values, namely numbers and text strings as well as the special
   values null, true, and false.
 
 Member:
-: A name/value pair in an object.  (A member is not itself a value.)
+: A name/value pair in an object.  (A member is not itself a JSON value.)
 
 Name:
-: The name in a name/value pair constituting a member.   (A name is not itself a value.) <!-- should we make it clear that names are (Unicode) strings? -->
+: The string name in a name/value pair constituting a member.   (A name is not itself a JSON value.)
```

Comment:
(1)
"string name" is a bit muddled.
Are there other names such as birth names :-)

(2)
That string, on its own, *is* a JSON value.
What we need to say is: for the purposes of JSONPath, we don't make this available as a node.